api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimDTO.java

```diff
@@ -28,4 +28,8 @@
     @ApiModelProperty(notes = "The fingerprint of the device used to make the claim, as best it can be identified. Probably a collection of header values.")
     private Map<String, Object> deviceFingerprint;
 
+    @JsonProperty("webUIVersion")
+    @ApiModelProperty(notes = "The version of the web-ui application used to create this request.")
```

Comment:
Should this mention that it is used in conjunction with proving which version of the Ts and Cs they accepted?

Reply:
done